data/darktableconfig.xml.in

```diff
@@ -1536,6 +1536,13 @@
     <shortdescription>expand a single darkroom module at a time</shortdescription>
     <longdescription>this option toggles the behavior of shift clicking in darkroom mode</longdescription>
   </dtconfig>
+  <dtconfig prefs="darkroom">
```

Comment:
As this option require another option to be set I would propose a combo box with multiple choices:

name :
**control expanded modules**

choices:

- no
- single visible module
- single module

With a tooltip with more explanations and a XML snippet for the documentation.

Reply:
Ok good idea

Reply:
Actually now I think about it I've remembered why I didn't do this. The problem is that the current darkroom/ui/single_module option can have its logic reversed by shift+clicking on a module header and that can't be easily achieved with a 3-way combobox. If you selected 'no' in the above combobox how should shift-click behave?

Reply:
I see! I didn't even know about the Shift-Click behavior :) So the PR is ok.